content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -236,10 +232,17 @@ sudo rmdir /mnt/data
 
 You can now close the shell to your Node.
 
+<!-- discussion -->
 
+## Mounting the same persistentVolume in two places
 
+{{< codenew file="pods/storage/pv-duplicate.yaml" >}}
 
-<!-- discussion -->
+You can perform 2 volume mounts on your nginx container:
+
+
+`/usr/share/nginx/html` for the static website
+`/etc/nginx/nginx.conf` for the default config
```

Comment:
These are meant to be bullet items, right? Or else they will be treated as a single paragraph.